src/appsco-billing-page.html

```diff
@@ -678,7 +678,9 @@ <h2 class="invoice-list-title">Invoice list</h2>
                 },
 
                 companyApi: {
-                    type: String
+                    type: String,
+                    value: function() { return ''; },
```

Comment:
This value assignment could be simple "value: ''" but this probably won't cause any problem.